website/docs/r/metrics_search.html.markdown

```diff
@@ -0,0 +1,101 @@
+---
+layout: "sumologic"
+page_title: "SumoLogic: sumologic_metrics_search"
+description: |-
+  Provides a Sumologic Metrics Search
+---
+
+# sumologic_metrics_search
+Provides a Sumologic Metrics Search.
```

Comment:
Add a link to help website about Metrics Searches.

Reply:
Just a note that log search markdown does not have a link